src/test/java/TestNGTests/CartTest.java

```diff
@@ -0,0 +1,54 @@
+package TestNGTests;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Optional;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+
+public class CartTest {
+    Cart andrewCart;
+    RealItem car;
+    VirtualItem disk;
+    SoftAssert softAssert;
+
+    @BeforeMethod(alwaysRun = true)
+    void init() {
+        andrewCart = new Cart("andrew-cart");
+        car = new RealItem();
+        softAssert = new SoftAssert();
+    }
+
+    @Test(groups = {"sanity"},priority = 2)
+    void totalPriceRealItem() {
+        car.setName("Audi");
+        car.setPrice(32026.9);
+        car.setWeight(1560);
+        andrewCart.addRealItem(car);
+
+        final double expected = andrewCart.getTotalPrice();
+        softAssert.assertEquals(expected, andrewCart.getTotalPrice());
+        softAssert.assertAll();
+    }
+
+    @Parameters({"name", "price","weight"})
+    @Test(groups = "regression", priority = 1)
```

Comment:
remove all the priorities, orders and depends

Reply:
I have removed priorities. Orders I quess are invocationCount (removed). At least I don't see any dependents in my code. I know that this is a dependency - dependsOnMethods. Or there are other established dependenies which I do not see.